test/backend.test.js

```diff
@@ -182,6 +182,16 @@ describe('Full walkthrough', function () {
           permissions: '*',
           allowedActions: '*',
           data: CSKp
+        },
+        {
+          id: creator.signingKeyId(),
+          name: 'creator',
+          purpose: ['sig'],
+          ringLevel: 1,
+          permissions: '*',
+          allowedActions: '*',
+          data: sbp('chelonia/rootState')[creator.contractID()]._vm.authorizedKeys[creator.signingKeyId()].data,
+          foreignKey: `sp:${encodeURIComponent(creator.contractID())}?keyName=${encodeURIComponent('csk')}`
```

Comment:
What is this, why was it added?

Reply:
I added a comment. TL;DR: Since now the validate function checks for signatures as part of its permission checking, we need to add an inner (unencrypted) signature here. It is good to also see that unencrypted inner signatures work.